ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/precompiles/privacy/PrivacyPrecompiledContract.java

```diff
@@ -187,19 +186,16 @@ public Bytes compute(final Bytes input, final MessageFrame messageFrame) {
           privateStateUpdater);
 
       final Bytes32 txHash = keccak256(RLP.encode(privateTransaction::writeTo));
-      final List<Log> logs = result.getLogs();
-      if (!logs.isEmpty()) {
-        privateStateUpdater.putTransactionLogs(txHash, result.getLogs());
-      }
-      if (result.getRevertReason().isPresent()) {
-        privateStateUpdater.putTransactionRevertReason(txHash, result.getRevertReason().get());
-      }
+      final Bytes blockHashTxHash = Bytes.concatenate(currentBlockHash, txHash);
```

Comment:
I think we should move this logic inside the privateStateUpdater. So the method would look like:
```
privateStateUpdater.putTransactionReceipt(blockhash, txHash, privateTransactionReceipt);
```

Reply:
Good idea, will do!